client/src/components/messenger/dialogs/AddConversationDialog.js

```diff
@@ -38,7 +38,8 @@ const AddConversationDialog = ({ onClose, contacts }) => {
                     />
                     <ul className="add_participants">
                         {contacts.map((contact) => (
-                            <li key={contact.contactId}>
+                            <li key={contact.contactId}
+                                title={contact.contactInfo.displayName + "#" + contact.contactId}>
                                 <div className="img_and_name">
                                     <img src={contact.contactInfo.imageUrl} alt="contact_img" />
                                     <span>{contact.contactInfo.displayName}</span>
```

Comment:
shouldn't this match the title?

`{contact.contactInfo.displayName + "#" + contact.contactId}`